src/features/burgerMenu/ui/burgerMenu.tsx

```diff
@@ -2,12 +2,137 @@ import { Menu } from "@headlessui/react";
 
 export function BurgerMenu() {
   return (
-    <Menu.Items>
+    <Menu.Items className="absolute flex flex-col right-0 z-[11] mt-10 w-[293px] h-[435px] px-10 py-[30px] rounded-[20px] bg-white shadow justify-start items-start gap-[15px] inline-flex focus:outline-none">
       <Menu.Item>
         {({ active }) => (
-          <a className={`${active && "bg-blue-500"}`} href="/account-settings">
-            <div className="bg-user-profile w-[24px] h-[24px]"> </div>
-            <p>Профиль</p>
+          <a className={`${active}`} href="/#">
```

Comment:
you need to add array data in /lib/consts and iterate this data here with .map().

Reply:
fixed